lib/client/checks.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 Gravitational, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"github.com/gravitational/teleport/api/client/okta"
+	"github.com/gravitational/teleport/lib/services"
+)
+
+// Compile time checks for the Okta client.
+var _ services.OktaImportRules = &okta.Client{}
+var _ services.OktaAssignments = &okta.Client{}
```

Comment:
`api` doesn't have access to these interfaces and it would create an import cycle since `api` is a submodule, so I'm not sure of where to put these. Any suggestions?